src/pages/User/mock.ts

```diff
@@ -0,0 +1,39 @@
+export type Subscriptions = {
+  __typename: 'subscriptions';
+  expiration: string;
+};
+
+export type MemberInformation = {
+  __typename: 'members';
+  first_names: string;
+  last_names: string;
+  email: string;
+  subscriptions: Subscriptions[];
+};
+
+export type UserType = {
+  __typename: string;
+  avatar_url?: string;
+  member_code: string;
+  member_info: MemberInformation[];
+};
```

Comment:
Please move these types to the types folder https://github.com/Azordev/did-app/blob/dev/src/utils/apollo.types.ts, just if that types doesn't exists yet, in case that the style already exist in the folder, please use it, you can modify it to add the properties that you needs